schemas/rum/error-schema.json

```diff
@@ -177,6 +177,90 @@
+                    "description": "Tells if the thread crashed.",
+                    "readOnly": true
+                  },
+                  "stack": {
+                    "type": "string",
+                    "description": "Unsymbolicated stack trace of the given thread.",
+                    "readOnly": true
+                  },
+                  "state": {
+                    "type": "string",
+                    "description": "Platform-specific state of the thread when its state was captured (CPU registers dump for iOS, thread state enum for Android, etc.).",
+                    "readOnly": true
+                  }
+                }
+              }
+            },
+            "binary_images": {
+              "type": "array",
+              "description": "Description of the each binary image (native libraries; for Android: .so files) loaded or referenced by the process/application.",
+              "items": {
+                "type": "object",
+                "description": "Description of the binary image (native library; for Android: .so file) loaded or referenced by the process/application.",
+                "required": ["uuid", "name", "is_system"],
+                "properties": {
+                  "uuid": {
+                    "type": "string",
+                    "description": "Build UUID that uniquely identifies the binary image.",
+                    "readOnly": true
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "Name of the library.",
+                    "readOnly": true
+                  },
+                  "is_system": {
+                    "type": "boolean",
+                    "description": "Determines if it's a system or user library.",
+                    "readOnly": true
+                  },
+                  "load_address": {
+                    "type": "string",
+                    "description": "Library's load address (hexadecimal).",
+                    "readOnly": true
+                  },
+                  "max_address": {
+                    "type": "string",
+                    "description": "Max from the library address range (hexadecimal).",
+                    "readOnly": true
+                  },
+                  "arch": {
+                    "type": "string",
+                    "description": "CPU architecture from the library.",
+                    "readOnly": true
+                  }
+                }
+              },
+              "readOnly": true
+            },
+            "was_truncated": {
+              "type": "boolean",
+              "description": "A boolean value saying if any of the stack traces was truncated due to minification.",
+              "readOnly": true
+            },
+            "meta": {
+              "type": "object",
+              "description": "Free-form metadata of the error event.",
+              "additionalProperties": true,
```

Comment:
I think it might make sense to explicitly list these attributes. ET FE depends on them for assembling the "Apple Crash Report" for _"download as"_. See more in: [Apple Crash Format Export (internal)](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/2977662661/Apple+Crash+Format+Export). It's worth having @maxep take on this.

Example of attributes we send here:
```json
    "error.meta": {
        "code_type": "arm64",
        "parent_process": "launchd_sim [31584]",
        "incident_identifier": "DAA6280F-F6A4-45C2-BA15-FC81DA0B6A4B",
        "process": "Example iOS [92909]",
        "exception_type": "SIGABRT",
        "exception_codes": "#0",
        "path": "/Application/121E89DC-4F9B-425D-B484-C6CC79FF3CA8/Example iOS.app/Example iOS"
    },
```

Reply:
That would be great, I have a full list [here](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/2977662661/Apple+Crash+Format+Export) but they are platform specific, I'm afraid having a very large set of metadata will be hard to manage. Maybe we should use a `oneOf` description and split the metadata for iOS and Android instead?